src/commands/force/lightning/lwc/test/run.ts

```diff
@@ -70,7 +70,10 @@ export default class Run extends SfdxCommand {
   }
 
   private runJest(args) {
-    return spawnSync(this.getExecutablePath(), args, { stdio: 'inherit' });
+    return spawnSync(this.getExecutablePath(), args, {
```

Comment:
I decided to keep the `spawnSync` here since we are interested in the status code to report back to the user. Adding the `shell` flag to make sure the node can be resolved.